helpers/github.ts

```diff
@@ -274,25 +274,23 @@ export async function writeTotalRewardsToGithub(totalRewards: number) {
     const filePath = "total-rewards.txt";
     const content = totalRewards.toString();
 
+    let sha: string | undefined = undefined; // Initialize sha to undefined
```

Comment:
```suggestion
    let sha: string; // Initialize sha to undefined
```

Reply:
Why are you initializing it as undefined?

Reply:
undefined is what octokit recommends, `string | null` throws an error
![Screenshot 2024-02-26 at 5 43 52 PM](https://github.com/ubiquity/devpool-directory/assets/42383942/5fb8be31-580d-4d2f-923b-e8bfd85db9ac)

Reply:
@pavlovcik

Reply:
> Okay let's follow the type safety of octokit.
> 
> let sha;
> 
> That's all you need.

Done